### ISSUE TITLE
Listing packages doesn't include those installed via `--git`

### ISSUE THREAD

**Issue by citrusmunch** [nabijaczleweli/cargo-update#245]
Not sure if this is a bug, intended behaviour, or just not yet implemented, but it would be nice to be able to include packages installed via `--git` in the listing to see if they require updates.

For example, after running `cargo install --git https://github.com/knorrfg/dotree`:

```ShellSession
❯ cargo install --list
cargo-update v13.2.0:
    cargo-install-update
    cargo-install-update-config
dotree v0.6.0 (https://github.com/knorrfg/dotree#c2fad7b9):
    dt
uiua v0.2.0:
    uiua
``` 

```ShellSession
❯ cargo install-update --list
    Polling registry 'https://index.crates.io/'..

Package       Installed  Latest   Needs update
cargo-update  v13.2.0    v13.2.0  No
uiua          v0.2.0     v0.2.0   No
``` 

There seems to be an attempt to print another table?

```ShellSession
❯ cargo install-update --list --git
    Polling registry 'https://index.crates.io/'..

Package       Installed  Latest   Needs update
cargo-update  v13.2.0    v13.2.0  No
uiua          v0.2.0     v0.2.0   No

Package  Installed  Latest  Needs update
```

**Comment by nabijaczleweli** [comment]
I'm assuming your `~/.cargo/.crates.toml` includes a line like
```toml
"dotree 0.6.0 (git+https://github.com/knorrfg/dotree#c2fad7b9c902f91208ec29869348b159db3762c9)" = ["dt"]
```
? Mine does and I can repro this, the git table is empty:
```
$ cargo install-update -lg
    Polling registry 'https://index.crates.io/'...............

Package        Installed  Latest   Needs update
b3sum          v1.3.1     v1.5.0   Yes
cargo-audit    v0.17.6    v0.18.3  Yes
cargo-bloat    v0.11.0    v0.11.1  Yes
cargo-deb      v1.35.0    v2.0.0   Yes
checksums      v0.6.0     v0.9.1   Yes
hyperfine      v1.14.0    v1.18.0  Yes
xargo          v0.3.24    v0.3.26  Yes
zoxide         v0.8.3     v0.9.2   Yes
bore           v0.4.1     v0.4.1   No
cargo-tree     v0.29.0    v0.29.0  No
cargo-update   v13.2.0    v13.2.0  No
gen-epub-book  v2.3.2     v2.3.2   No
openalias      v0.2.0     v0.2.0   No
pixelmatch     v0.1.0     v0.1.0   No
treesize       v0.5.0     v0.5.0   No

Package  Installed  Latest  Needs update

```

**Comment by nabijaczleweli** [comment]
Thanks, released in [`v13.2.1`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v13.2.1)
